components/02-elements/action/Action.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="action">
```

Comment:
Meh, this component in "old" Alpaca obey rules, b/c don't act like an element, but in reality, it's a wrapper that creates a layout for buttons and links.

Let's for now just put it on some "to do" list (idk, maybe some separate directory?) (probably most of the dashboard related stuff will need some love), we will back to it later.